hash-store/src/main/java/com/scalar/dl/hashstore/client/tool/Bootstrap.java

```diff
@@ -18,6 +19,7 @@ public static void main(String[] args) {
   @Override
   protected Integer execute(HashStoreClientService service) throws ClientException {
     service.bootstrap();
+    Common.printOutput(null);
```

Comment:
Just curious, what's this for?

Reply:
@choplin Thanks for the question! It shows the following result. We have shown it in the registration commands, probably for users' peace of mind, and I followed it.

```console
{
  "status_code" : "OK",
  "output" : null
}
```